modules/standard/LinkedLists.chpl

```diff
@@ -220,7 +220,7 @@ record LinkedList {
     Returns the data stored in the first element of the list
     It is an error to call front() on an empty list
    */
-  proc front() {
+  proc front(): eltType throws {
```

Comment:
By adding `throws` I also meant to add tags to the [documentation](https://github.com/chapel-lang/chapel/blob/master/modules/standard/Path.chpl#L420). Sorry, to be unclear. I am also not very sure about what error type it would.